newrelic/hooks/mlmodel_sklearn.py

```diff
@@ -80,12 +80,52 @@ def _nr_wrapper_method(wrapped, instance, args, kwargs):
         if method in ("predict", "fit_predict"):
             training_step = getattr(instance, "_nr_wrapped_training_step", "Unknown")
             wrap_predict(transaction, _class, wrapped, instance, args, kwargs)
+            create_label_event(transaction, _class, instance, return_val)
             return PredictReturnTypeProxy(return_val, model_name=_class, training_step=training_step)
         return return_val
 
     wrap_function_wrapper(module, "%s.%s" % (_class, method), _nr_wrapper_method)
 
 
+def create_label_event(transaction, _class, instance, return_val):
+    inference_id = uuid.uuid4()
```

Comment:
This uuid needs to be the same as the feature event uuid since they are part of the same prediction.